### ISSUE TITLE
Likely original source, and potential license violation

### ISSUE THREAD

**Issue by mk-pmb** [thedava/fast-leaf-decay#1]
tl;dr: I invite you to dispose of the current questionable upstream and restart the project based on FLD from VanillaTweaks and in accordance with their license.

Hi! You may have been scammed when you downloaded it from planetminecraft.com. I was checking for a proper license because your readme didn't give any, so I followed your link https://www.planetminecraft.com/data-pack/fast-leaf-decay-the-best/ and from there tried to visit the project website link, which was a redirect to [⚠ Potential Malware ahead! ⚠ AdFocus 68596180688223](http://adfoc.us/68596180688223). Investigating the shady nature of the AdFocus, I read that while it has legitimate uses, lots of people would use it to make a quick buck off other people's work.

Fortunately, whoever repackaged it has left the description in `pack.mcmeta`, where it says `vanillatweaks.net`. So it's probably the FLD from https://vanillatweaks.net/picker/datapacks/ . They do indeed have [a license](https://vanillatweaks.net/terms/) which requires proper attribution. You may or may not have a custom license that would allow distribution in the current form; however, for collaboration on updates, it would be easier to have the code stem from a reliable source using a general audience license.

**Comment by mk-pmb** [comment]
Restarting the project would also be a good opportunity to add the missing final newline characters to all text files that don't have them, to get rid of all the `\ No newline at end of file` messages in the git diffs.

**Comment by thedava** [comment]
Well, tbh I didn't knew the original source of this package. I found it on planetminecraft, I added compatibility for 1.19 and thats it. I haven't had a look into any other file tbh. Thank you for researching the original author. As far as I can see their package is up2date with MC 1.19 so there is no need for this repository anymore. I will adjust the README and archive this project
